src/Components/Bookmark.jsx

```diff
@@ -0,0 +1,150 @@
+import React,{useEffect, useState} from "react";
+import "../assets/css/Bookmark.css";
+
+
+function DailyForecast({day, forecast}) {
+  return (
+      <>
+      <div className='daily-container'>
+          <div className='day'>{day}</div>
+          <img src={"http://openweathermap.org/img/wn/"+forecast.icon+"@2x.png"} alt="cloudy-icon"/>
+				<p>{Math.round(forecast.temperature.maximum)}&#8451;</p>
+				<p>{Math.round(forecast.temperature.minimum)}&#8451;</p>
+      </div>
+      </>
+  )
+}
+
+function Bookmark({x, weather, city, dailyforecast}) {
+	const days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
+	return (
+		<>
+		<div className="bookmark">
+			<div className="bookmark-country-name">
+				<h1>{city}</h1>
+			</div>
+			<div className="bookmark-cloud-type">
+				<div className='weather-icon'>
+					<img src={"http://openweathermap.org/img/wn/"+dailyforecast[0].icon+"@2x.png"} alt='cloudy_img'/>
+        </div>
+				<p>{dailyforecast[0].condition}</p>
+			</div>
+			<div className="bookmark-wind-humidity">
+				<ul>
+					<li> Wind {dailyforecast[0].windspeed}m/s </li>
+					<li> Humidity {dailyforecast[0].humidity}% </li>
+				</ul>
+			</div>
+			<div className="bookmark-temp">
+				<h1>{ Math.round(dailyforecast[0].current_temp) }&#8451;</h1>
+			</div>
+			<div className="flex bookmark-daily-forecast">
+		    {dailyforecast.map((forecast, index) => (
```

Comment:
@joshi-kaushal 
`forecast` does not have a unique identifier key :(. I renamed `index` to `id` though(hope that's better).

Reply:
Using array index is a bad react practice since it changes with every array updation and may cause rerender. But since you don't have `id` in your data, you can use it here.

Keep it as it is, for now, it's fine.

If you're more interested:
- [React JS documentation](https://reactjs.org/docs/lists-and-keys.html#keys) 
- [Index as a key is an anti-pattern
](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318)

> When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort. -- react docs